### ISSUE TITLE
Deposit ETH but not credit to Etherdelta balance

### ISSUE THREAD

**Issue by cryptomen** [etherdelta/etherdelta.github.io#111]
This is my first trying out Etherdelta and I would need to fund it. After reading the guide and youtube videos, made a first transaction of 0.01 ETH, it went through and confirmed by etherscan.io but not reflected in etherdelta eth wallet. So I went for a second try and made another transaction 0.001 ETH and this time it also went through and credited to etherdelta balance. So now I can only see 0.001 in the balance. My Ether Wallet is less 0.01 and 0.001 ETH though. So I do not know where the first transaction went to and what happen? Anyone able to help?

**Comment by zackcoburn** [comment]
Link me to the transactions on Etherscan.

Zack Coburn

NOTICE: This e-mail transmission (and/or the attachments accompanying it)
may contain confidential information belonging to the sender which is
protected by law. The information is intended only for the use of the
intended recipient. If you are not the intended recipient, you are hereby
notified that any disclosure, copying, distribution or the taking of any
action in reliance on the contents of this information is strictly
prohibited. Any unauthorized interception of this transmission is illegal.
If you have received this transmission in error, please promptly notify the
sender by reply e-mail, and then destroy all copies of the transmission.

**Comment by cryptomen** [comment]
https://etherscan.io/tx/0x680bc108de369a6fb12096f5f99dc913e4cc7f67b117ec23d74c76fdd50cdbc4

**Comment by zackcoburn** [comment]
I'm not sure how you got into this situation, but your first deposit went
to an old EtherDelta smart contract. You can get it back easily:
1. Go into the "Smart contract" menu at the top.
2. Choose the "0x373..." one (marked as OLD).
3. You'll see your funds there.
4. Use the Withdraw form to withdraw.
5. Switch back to the CURRENT smart contract and deposit.

Zack Coburn

NOTICE: This e-mail transmission (and/or the attachments accompanying it)
may contain confidential information belonging to the sender which is
protected by law. The information is intended only for the use of the
intended recipient. If you are not the intended recipient, you are hereby
notified that any disclosure, copying, distribution or the taking of any
action in reliance on the contents of this information is strictly
prohibited. Any unauthorized interception of this transmission is illegal.
If you have received this transmission in error, please promptly notify the
sender by reply e-mail, and then destroy all copies of the transmission.

**Comment by cryptomen** [comment]
hi zack, thanks so much. Mystery solved. I saw the 0.01 in the balance in the old contract.
So to withdraw to wallet, can I withdraw all 0.01?
I actually submitted a request and it's still pending now and the value of the transaction it states 0 ether. You can check the etherscan for the address above.

**Comment by zackcoburn** [comment]
Yep, you can withdraw the whole thing. Ethereum is pretty bogged down right
now with pending transactions, so it might take a little while. The value
being 0 is normal -- the important stuff is in the transaction's data,
which basically says "hey EtherDelta smart contract, move the 0.01 back to
my wallet, and I'm sending 0 ETH with this transaction by the way."

Zack Coburn

NOTICE: This e-mail transmission (and/or the attachments accompanying it)
may contain confidential information belonging to the sender which is
protected by law. The information is intended only for the use of the
intended recipient. If you are not the intended recipient, you are hereby
notified that any disclosure, copying, distribution or the taking of any
action in reliance on the contents of this information is strictly
prohibited. Any unauthorized interception of this transmission is illegal.
If you have received this transmission in error, please promptly notify the
sender by reply e-mail, and then destroy all copies of the transmission.

**Comment by cryptomen** [comment]
sure, not in a hurry though. I will check it in a day. Thanks much for the spot on! I can safely use etherdelta moving forward.

**Comment by chetan-pandit** [comment]
Hello Mate.. I am facing a similar issue and i have used the latest contract as well. Any solution will be much appreciated. 
txn hash -- 0x16ab2a9d26afd19807f5362d3f24921405b652d2fd60fa98c8e2e280b972fc8c

**Comment by benjoun** [comment]
Hi! Me too for at least a week, the deposit on etherdelta doesn't appear after the transaction has indicated success for a week now. If someone knows a solution please let a message or contact us. 

txn hash -- 0x7dafd86695c5f03121ec574aed39a12c014185b79ced571210d8334817644509do)
Thanks for your help
